packages/cli/medusa-cli/src/commands/new.ts

```diff
@@ -187,7 +187,7 @@ const clone = async (hostInfo, rootPath, v2 = false, inputBranch) => {
       : []
 
   if (v2) {
-    branch = [`-b`, inputBranch || "feat/v2"]
+    branch = [`-b`, inputBranch || "master"]
```

Comment:
We don't need this option anyway so better to remove this.

Reply:
The `inputBranch`?

Reply:
The `v2` condition, we don't need a `--v2` option anymore

Reply:
I noticed this is a more comprehensive change, so I'll save this cleanup for later.